### PR TITLE
Fix: Remove manual deployment README and cleanup root and infra READMEs.

### PR DESCRIPTION
**Pull request by michaelwsherman**
This should complete the docs cleanup for 2.1.

- Manual deployment README deleted and no longer linked from root README. b/358367459
- Instructions on adding dashboards added into root README from manual deployment README and rewritten.
- Root README deployment section only points to terraform deploy. b/363338347
- A bit more info about frontend and backend deployment. b/363336571
- Add a link to Known Errors at the top of Deployment guide. b/369208289
- Deployment manager enablement in deployment prerequisites. b/367779528
- Made a pass of READMEs after cleaning everything above up. b/358367615

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -3,23 +3,28 @@ Terraform simplifies deploying Generative AI for Marketing. The Terraform deploy
 
 *Note*: The Terraform Provider for Google Cloud is not able to generate some of the GenAI resources, `null_resource` is used to create some resources using the Google Cloud SDK.
 
-You'll need to create a Google Cloud project and link a billing account before you begin. **It is strongly recommended you deploy Generative AI for Marketing in it's own fresh project.** Existing resources in a project may be impacted by the deployment, and the deployment itself may fail.
+You'll need to create a Google Cloud project and link a billing account before you begin. **It is strongly recommended you deploy Generative AI for Marketing in its own fresh project.** Existing resources in a project may be impacted by the deployment, and the deployment itself may fail.
```
Comment (ghchinoy) [inline infra/README.md:6]: "... in its own, new project."

File: infra/README.md [hunk 2]
```diff
@@ -3,23 +3,28 @@ Terraform simplifies deploying Generative AI for Marketing. The Terraform deploy
...
+If you encounter problems during deployment see the [Known Issues](#known-issues) section for workarounds to common issues.
+
 ## Step 0 - Prerequisites:
 
 Before executing Terraform, follow these steps to enable some services:
 
-### Get allowlisted for Imagen3
-Request access to Imagen3 through this [form](https://docs.google.com/forms/d/e/1FAIpQLSdMHAK_KJygnvV2Psga7FIzKAhAqIBS_bHYzfgf_Y2h7fsoGA/viewform).
+### Get Allowlisted for Imagen3
+Request access to Imagen3 through this [form](https://docs.google.com/forms/d/e/1FAIpQLSdMHAK_KJygnvV2Psga7FIzKAhAqIBS_bHYzfgf_Y2h7fsoGA/viewform). Note this can take up to a week. You can still use Generative AI for Marketing while awaiting allowlisting, but image generation capabilities will not work.
```
Comment (ghchinoy) [inline infra/README.md:21]: 'Imagen 3" (space between "Imagen" and the version number "3')

File: infra/README.md [hunk 3]
```diff
@@ -3,23 +3,28 @@ Terraform simplifies deploying Generative AI for Marketing. The Terraform deploy
...
+
+### Enable the Cloud Resource Manager API:
+1. Go to https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview and enable the API.
 
 ### Enable Firebase
-Will be used for the frontend deployment
+The frontend of Generative AI for Marketing runs in Firebase. Before beginning deployment, you need to enable Firebase.
```
Comment (ghchinoy) [inline infra/README.md:27]: Consider "is hosted on" rather than "runs in"

**Review by ghchinoy — CHANGES_REQUESTED** [review]
Minor comments

File: infra/README.md (hunk 1 above)
Comment (michaelwsherman) [reply infra/README.md:6]: fixed

File: infra/README.md (hunk 3 above)
Comment (michaelwsherman) [reply infra/README.md:27]: fixed

File: infra/README.md (hunk 2 above)
Comment (michaelwsherman) [reply infra/README.md:21]: fixed.

**Review by ghchinoy — APPROVED** [review]
LGTM

**Pull request merged**